owslib/wps.py

```diff
@@ -1077,9 +1077,13 @@ def _parseLiteralData(self, element, literalElementName):
             for sub_element in literal_data_element:
                 subns = getNamespace(sub_element)
                 if sub_element.tag.endswith('DataType'):
-                    reference = sub_element.get(nspath("reference", ns=subns)) or sub_element.text
-                    if reference and ':' in reference:
-                        self.dataType = reference.split(':')[-1]
+                    self.dataType = sub_element.text
+                    if not self.dataType:
+                        reference = sub_element.get(nspath("reference", ns=subns))
+                        pos = len(reference) - 1
+                        while pos >= 0 and reference[pos].isalpha():
+                            pos -= 1
+                        self.dataType = reference[pos + 1:]
```

Comment:
Add comment "get substring after # or : for type: string, etc"

Reply:
done in 02a3b66